code/game/gamemodes/scores.dm

```diff
@@ -209,7 +209,7 @@ GLOBAL_VAR_INIT(score_technomancer_faction_item_loss, 0)
 	//NeoTheology Modifiers
 	GLOB.score_neotheology_faction_item_loss -= GLOB.neotheology_faction_item_loss * 150 //300
 	GLOB.neotheology_objectives_score = GLOB.neotheology_objectives_completed * 25 // ~100
-	GLOB.score_mess -= GLOB.dirt_areas * 25 //~250
+	GLOB.score_mess -= GLOB.dirt_areas * 10 //~250
```

Comment:
Shouldn't comment be updated as well?

Reply:
This is not from my PR but due to commit changes to make the changes suitable for the test server, it has been removed.